internal/buildscripts/update-dep

```diff
@@ -11,10 +11,11 @@ fi
 # If MODULE is "go.opentelemetry.io/collector" need to update "go.opentelemetry.io/collector/model"
 # and "go.opentelemetry.io/collector/pdata" as well.
 if [ "$MODULE" == "go.opentelemetry.io/collector" ]; then
-  MODEL=$MODULE"/model"
-  if grep -q "$MODEL " go.mod; then
-    go get "$MODEL"@"$VERSION"
-  fi
+  # TODO: Uncomment this after semconv releases.
```

Comment:
Reason for this commented code is here https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9394